evap/staff/tests/test_views.py

```diff
@@ -2229,16 +2230,33 @@ def test_questionnaire_with_answers_warning(self):
             '<label class="form-check-label badge bg-danger" for="id_contributions-1-questionnaires_0">', page
         )
 
-    @patch.dict(Evaluation.STATE_STR_CONVERSION, {Evaluation.State.PREPARED: "mock-translated-prepared"})
     def test_state_change_log_translated(self):
```

Comment:
The new test does not contain any `assertNotIn`, so it is very well possible that "prepared", "new", etc. come up accidentally. Do you think you can come up with a way to avoid this, or is it too annoying now that Django handles the translation?

Reply:
(I think the better translation is worth more than the test here, so I am not expecting any changes outside of tests for this comment)